projects/sio2/sio2.js

```diff
@@ -0,0 +1,99 @@
+const sdk = require("@defillama/sdk");
+const abi = require("./abi.json");
+const { default: BigNumber } = require("bignumber.js");
+const addressesProviderRegistry = "0x9D8bB85b1c728f69672923dD4A0209EC8b75EFda"; // from https://github.com/SiO2-Finance/sio2-protocol/blob/main/deployed-contracts.json
```

Comment:
Let's change the comment to refer to our documentation: https://sio2-finance.gitbook.io/en/development/contract-addresses